lib/scrabble.rb

```diff
@@ -1,11 +1,9 @@
+require 'pry'
 class Scrabble
+  attr_reader :values
 
-  def score(word)
-    1
-  end
-
-  def point_values
-    {
+  def initialize
+    @values = {
```

Comment:
Recommend you leave this as a method call. Not positive what the speed implications are (would need to benchmark it), but try to get used to stepping into an unfamiliar code base and using the tools that have been made available to you.